src/potentiometer-element.ts

```diff
@@ -0,0 +1,269 @@
+import { customElement, html, LitElement, property } from 'lit-element';
+
+interface Point {
+  x: number;
+  y: number;
+}
+
+let id = 1;
+
+/** The potentiometer SVG is taken from https://freesvg.org/potentiometer and some of the
+    functions are taken from https://github.com/vitaliy-bobrov/js-rocks knob component */
+@customElement('wokwi-potentiometer')
+export class PotentiometerElement extends LitElement {
+  @property() min = 0;
+  @property() max = 100;
+  @property() value = 0;
+  @property() step = 1;
+  @property() startDegree = -135;
+  @property() endDegree = 135;
+  private center: Point = { x: 0, y: 0 };
+  pressed = false;
+  id = `rotating-${id++}`;
+
+  clamp(min: number, max: number, value: number): number {
+    return Math.min(Math.max(value, min), max);
+  }
+
+  mapToMinMax(value: number, min: number, max: number): number {
+    return value * (max - min) + min;
+  }
+
+  percentFromMinMax(value: number, min: number, max: number): number {
+    return (value - min) / (max - min);
+  }
+
+  render() {
+    return html`
+      <style>
+        #${this.id} {
+          transform-origin: -120.168198px -140.5804px;
+          transform: rotate(var(--knob-angle, 0deg));
+        }
+      </style>
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        width="20mm"
+        height="20mm"
+        viewBox="0 0 20 20"
+        @mousedown=${this.down}
+        @mousemove=${this.move}
+        @mouseup=${this.up}
+        @touchstart=${this.down}
+        @touchmove=${this.move}
+        @touchend=${this.up}
+      >
+        <g transform="translate(-6.7420057,-6.6693388)">
```

Comment:
Can you remove these transforms?

This is how I do it: open the SVG file in inkscape, select the group (you can simply click on one of the elements, or press Ctrl+Shift+X to open the XML explorer, and then select it from the XML explorer), and then ungroup (Ctrl+U). 

Ungrouping removes these transforms, and changes the x/y (and possibly heigh/width) of the child elements, so that they already have the transforms applied.

Reply:
Also, make sure to do this before doing any of the other SVG changes I suggested (removing styles, tspans, etc.), because inkscape is likely to add them again

Reply:
Done